gits/terminal.py

```diff
@@ -720,3 +704,27 @@ def dumphtml(self):
                 span += '\n'
 
         return r
+
+    #
+    # User visible methods.
+    #
+    def generate_html(self, buf):
+        """Determines the terminal commands. Then the commands are executed.
+        Then the html, representing the state of the terminal after the
+        commands executing, is generated. Commands are either the control
+        characters or the escape sequences, describing in on of the
+        configuration files. """
```

Comment:
I liked the whole idea, but the docstring needs to be slightly reworked. First of all, it doesn't tell anything about text which doesn't belong to either escape or control sequences. The text is printed on the screen as is. Then, the docstring doesn't contain any information about the `buf` parameter. I think it's very important to say that `generate_html` takes bytes obtained from a terminal-oriented program. Finally, there must be some information about the result of the routine i.e. HTML.